index.html

```diff
@@ -0,0 +1,116 @@
+                </ul>
+            </div>
+          </nav>
+          <section id="FirstPage">
+            <div class="Planets container">
+                <ul type="button" class="hamburger col-1 col-sm-1">
+                    <li class="HamLine1"></li>
+                    <li class="HamLine2"></li>
+                    <li class="HamLine3"></li>
+                </ul>
+                <div class="nav_top">
+                    <p class="Call col-lg-5 col-8 col-sm-8">"Wake up Tenno!"</p>
+                    <h1 class="IntroText col-lg-10 col-12 col-sm-12">Top six warframes chosen by the community in 2022</h1>
+                    <p class="Reflect col-lg-6 col-11 col-sm-11">Looking back this year on what warframe where the most popular, and looking at why they were placed the way they are.</p>
+                    <p class="Date col-lg-5 col-12 col-sm-12">01 January 2022 - 31 December 2022</p>
+                    <p class="Host col-lg-6 col-12 col-sm-12">Ordis is...in despair...PLEASED, to host this!</p>
+                </div>
+            </div>
+          </section>
+          <section id="Judgements">
+            <div class="Program">
+                <h3 class="Aspects col-lg-4 offset-lg-4 col-8 col-sm-8 offset-sm-2 offset-2">Aspect that each warframe was judged on</h3>
+                <div class="col-lg-2 offset-lg-5 col-2 col-sm-2 offset-sm-5 offset-5"><hr class="Line col-lg-2 offset-lg-5 col-2 col-sm-2 offset-sm-5 offset-5"></div>
+                <li class="Missions col-lg-12"><ul class="row col-lg-11">
+                    <li class="Judge col-lg-2 offset-lg-1"><image class="Images col-lg-5" src="images/Eidolon.png"></image><h2 class="col-lg-12">Eidolon Hunts</h2><p class="col-lg-12 col-1 col-sm-1">Ordis has..stalked you ...ANALIZED the operators hunts. Oh if only Ordis had a body.</p></li>
+                    <li class="Judge col-lg-2"><image class="Images col-lg-5" src="images/vitus.png"></image><h2 class="col-lg-12">Arbitration</h2><p class="col-lg-12">Ordis thinks that the operator has ...wasted time ...INVESTED a lot in farming vitus essence.</p></li>
+                    <li class="Judge col-lg-2"><image class="Images col-lg-5" src="images/normal.png"></image><h2 class="col-lg-12">Normal Chart</h2><p class="col-lg-12">The Lotus must be so proud of the operator, maybe Ordis will meet a similar fate someday.</p></li>
+                    <li class="Judge col-lg-2"><image class="Images col-lg-5" src="images/steel.png"></image><h2 class="col-lg-12">Steelpath</h2><p class="col-lg-12">Operator we can not let Teshin be forgotten. QUITE NOW ...Carry out his dream.</p></li>
+                    <li class="Judge col-lg-2"><image class="Images col-lg-5" src="images/archon200.png"></image><h2 class="col-lg-12">Archon Hunts</h2><p class="col-lg-12">Ordis thinks the operator looks ...like a porcupine ...SPLENDID with the shards installed.</p></li>
+                </ul></li>
```

Comment:
- [ ] It is not semantically correct to wrap a `ul` tag inside an `li` tag, it is supposed to be the other way round, kindly fix this.